### DISCUSSION TITLE
Issue with white screen in snap camera app

### DISCUSSION THREAD

**Discussion by CrazyCreepy** [ptrumpis/snap-camera-server#71] [Q&A] [answered]
Hi after the installation the Video feedback ain't working and now my camera is use by someone and don't give a feed back anymore.
I guess I have done something wrong with my manipulation or I miss a step. [Here](https://github.com/ptrumpis/snap-camera-server/tree/main#-snap-camera-server-v3) is the instruction that I follow and I also use [this video](https://youtu.be/bcsjvWHUr7c)
to help me with te step.  For your Information(FYI), I have no experience with docker I "understood" the concepte but it's the first time I use it. So here the first clue of where it could go wrong.

So now I have Docker with 4 element in it and can run it. I'm able to run Snap-camera-server, but the feedback camera is empty and
I don't have any lens. 

The green led on my camera tell me that a software is using it. 
The red led on my camera tell me that it power on.

 ###  here the result:
![image](https://github.com/ptrumpis/snap-camera-server/assets/63480822/b89917d6-1a57-498a-85df-df3b1aecf0fc)

I don't know if it's the snap-cam environnement that use it on another software. But now no one can use the Camera.

any help will be appreciate.

(TL;DR) I done all the installation proccess like in the tutorial, but I have no camera feedback, no lens filter and now my camera is lock.

**Comment by ptrumpis** [comment]
There is no problem with the docker server. The server is working fine (according to screenshots and logs).

You can try to call https://studio-app.snapchat.com/ in your browser to verify.

You need to check your snap camera application settings in the top right corner or try to force quit snap camera application and restart.

**Comment by CrazyCreepy** [comment]
Oh! I have already done it, I just forgot to mention it. 
### Here the result
![image](https://github.com/ptrumpis/snap-camera-server/assets/63480822/c76e6d47-13f5-4df0-b097-b1b6848e052b)

**Reply by ptrumpis** [reply, marked as answer]
So did you force kill `Snap Camera.exe` in your Task Manager?

1. You need to start the server before you start the application.
2. If you start the application before the server, you will get a blank screen.
3. If you close snap camera window it will continue to run in the background.
4. You need to right click the tray icon in your bottom right corner and select "Quit" or force kill with your Task Manager to fully close the application.
5. Try to re-open snap camera after fully closing and it should connect to the server.

**Comment by CrazyCreepy** [comment]
I restart my Computer and I also close the snap camera that start at the star up and it's now working
